### ISSUE TITLE
[Question]: All address lookup tentatives failed

### ISSUE THREAD

**Issue by robinhood-code** [bropat/eufy-security-ws#290]
### Ask your question

I still have this issue everyday for a particular device and happen almost everyday at the same time. (~4:45am)

I have not been able to find out what exactly had happened at this time which would have caused the device failed. I don't have any automation being triggered at that particular point of time that I can identify. 

Once I've got this error, the settings of the device cannot be changed by HA anymore, and I have to restart my eufy-security-ws container to get the device running again. Therefore I have currently a daily schedule to restart the container at 6:00pm everyday.

I have 2 questions,

1. is it true that once the error "All address lookup tentative failed" is encountered, eufy-security-ws does not try to connect to the device anymore and have to do a container / service restart to get things up and running again? 

2. Is there anything I can do to help find out what exactly happen at ~4:45am everyday to trigger this the "All address lookup tentatives failed" error? 

Thanks.

**Comment by PhilippEngler** [comment]
Try to answer your questions:

1. The client which eufy-security-ws uses of course is trying to reconnect to the devices where the connection is lost. Have you multiple devices (multiple HomeBases, Indoor-/SoloCams) to which a p2p connection is build up?
2. Around the mentioned time, the P2P connection to devices seems to be run in an timeout. I use multiple devices (2 HomeBases and 1 IndoorCam) and the connection is randomly disconnection for at least one device, but the connection is every time restored.

Does the `All address lookup tentatives failed` error is only logged once or repeating for the device?

**Comment by robinhood-code** [comment]
So I managed to do a bit more trials and errors. There was an WIFI option on my router to disable "Wifi PMF". Once I've disabled this apparently I no longer have disconnection issue with eufy-security-ws. (Connection was never an issue with the offical apps thought) 

I still have issue with eufy-security-ws thought, where every few days my HA Eufy integration does not respond to commands. Again once I have restarted eufy-security-ws, everything works again so I still keep the daily eufy-security-ws restart. 

This time however, when I check the eufy-security-ws logs, there was no error message anymore and in HA the device is still "connected"
